### ISSUE TITLE
Update URLs for downloading Chelsa 21k dataset

### ISSUE THREAD

**Issue by kpmercier** [EvolEcolGroup/pastclim#59]
I am trying to use PastClim to download the Chelsa 21k dataset but get the error below. 

It seems like the URL where Chelsa stores the data has changed. For example, for bio01 0k years ago:

URL that PastClim is trying to use-
https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_0_V1.0.tif

URL from chelsa website-
https://os.zhdk.cloud.switch.ch/chelsav1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_0_V1.0.tif

```r
> library("pastclim")
Loading required package: terra
terra 1.7.83
> get_available_datasets()
Barreto2023, Beyer2020, CHELSA_trace21k_1.0_0.5m_vsi, Example, HYDE_3.3_baseline, Krapp2021, paleoclim_1.0_10m, paleoclim_1.0_2.5m, paleoclim_1.0_5m
for present day reconstructions, use "WorldClim_2.1_RESm" or "CHELSA_2.4_RESm" where RES is an available resolution.
for future predictions, use "WorldClim_2.1_GCM_SSP_RESm" or "CHELSA_2.1_GCM_SSP_RESm", where GCM is the GCM model, SSP is the Shared Socio-economic Pathways scenario.
use help("WorldClim_2.1") or help("CHELSA_2.1") for a list of available options
> download_dataset("CHELSA_trace21k_1.0_0.5m_vsi")
Error in sf::gdal_utils(util = "buildvrt", source = chelsa_url, destination = filename,  : 
  gdal_utils buildvrt: an error occured
In addition: There were 50 or more warnings (use warnings() to see the first 50)
> warnings()
Warning messages:
1: In CPL_gdalbuildvrt(if (missing(source)) character(0) else source,  ... :
  GDAL Message 1: Can't open /vsicurl/https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_20_V1.0.tif. Skipping it
2: In CPL_gdalbuildvrt(if (missing(source)) character(0) else source,  ... :
  GDAL Message 1: Can't open /vsicurl/https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_19_V1.0.tif. Skipping it
3: In CPL_gdalbuildvrt(if (missing(source)) character(0) else source,  ... :
  GDAL Message 1: Can't open /vsicurl/https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_18_V1.0.tif. Skipping it
4: In CPL_gdalbuildvrt(if (missing(source)) character(0) else source,  ... :
  GDAL Message 1: Can't open /vsicurl/https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V1/chelsa_trace/bio/CHELSA_TraCE21k_bio01_17_V1.0.tif. Skipping it
[...]
```

```r
> utils::sessionInfo()
R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] pastclim_2.1.0 terra_1.7-83  

loaded via a namespace (and not attached):
 [1] digest_0.6.37      codetools_0.2-20   fastmap_1.2.0      xfun_0.48         
 [5] magrittr_2.0.3     e1071_1.7-16       KernSmooth_2.23-24 knitr_1.48        
 [9] htmltools_0.5.8.1  rmarkdown_2.28     classInt_0.4-10    sf_1.0-18         
[13] cli_3.6.3          grid_4.4.1         DBI_1.2.3          proxy_0.4-27      
[17] class_7.3-22       compiler_4.4.1     rstudioapi_0.16.0  tools_4.4.1       
[21] evaluate_1.0.1     Rcpp_1.0.13        yaml_2.3.10        rlang_1.1.4       
[25] units_0.8-5
```

**Comment by dramanica** [comment]
Thank you @kpmercier for reporting this. It looks like the changes also impact the other CHELSA datasets. I'll try to put out a fix ASAP.

**Comment by dramanica** [comment]
Now fixed in 79a372d5bb717a330f86a2f7a18d1daa10603a63, on the `dev` version. As we now explain on the landing page of the website, you will need to install the `dev` version with:
`install.packages("pastclim", repos = c("https://evolecolgroup.r-universe.dev")`
Thanks @kpmercier for reporting the problem.
